charts/etcd/templates/etcd-compaction-cronjob.yaml

```diff
@@ -25,6 +25,11 @@ metadata:
 spec:
   schedule: {{ .Values.backup.backupCompactionSchedule }} 
   concurrencyPolicy: Forbid
+{{- if eq (int $.Values.replicas) 0 }}
```

Comment:
Might be better to check [statefulsetReplicas](https://github.com/gardener/etcd-druid/blob/1ae26eeb63e742cce6c4ca430d09593f51eaf7f0/charts/etcd/templates/etcd-statefulset.yaml#L22).